app/models/merchant.rb

```diff
@@ -1,4 +1,13 @@
 class Merchant < ActiveRecord::Base
   validates_presence_of :name
+
   has_many :items
+
+  def self.how_many_items?
```

Comment:
Were you working on merchants as well?

Reply:
I wasn't working on merchants, but it looks like I might have added a space in between.